Jenkinsfile

```diff
@@ -39,11 +40,6 @@ pipeline {
 
         // Docker build flags are set via the getDockerBuildFlags() shared library.
         sh "docker build ${getDockerBuildFlags()} -t rightbrainnetworks/auto-semver:${env.VERSION} ."
-
-
-        sh "python setup.py sdist"
```

Comment:
If you remove this how are you getting the package out of the docker image after the docker image build?

Reply:
Within Docker image